packages/interpolate-components/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Releases
```

Comment:
super nit: We could follow the template https://keepachangelog.com/en/1.0.0/. I personally like it adds a heading level with `Added`, `Removed`...

That being said, most of our Changelogs don't follow it anyway, so feel free to ignore :)

Reply:
I'll try to find some time and port existing changelogs to that format, so at least it is unified across the whole repo.

Reply:
You're right and I agree with you. I'll go the extra _extra_ mile and using that now that my tests have finally passed and I could easily ship this 😆